ISSUES.md

```diff
@@ -0,0 +1,55 @@
+# Issues
+
+## How to create new issues properly
+
+GitHub issues are used for everything from bug reporting to long-term ideas. As such you can make everything much smoother by following some simple rules.
+
+### Rule zero
+
+Always try to give your issue a meaningful title as this is the first thing anyone will see.
+
+Note: `[CR]` and `[WIP]` "tags" are meaningful only for PRs. All open issues by definition are request for comments and work in progress.
+
+### Bug reports
+
+Before you submit a bug always search the issues to see if it hasn't been reported already.
+
+Your bug report has to include:
+
+- On what OS did you experience the problem (Windows, Linux, OS X etc.)
+- What version were you playing:
+ - Tiles or Curses (text-based)
+ - Version string (preferably full version e.g. "0.C-4547-g3f1c109", or Jenkins build number e.g. 3245)
+- Description of the problem you've found written in a way that enables anyone to try recreate it
+
+Your bug report may include:
+
+- Screenshot(s) as some things are best explained visually
+- Save file (e.g. link to a dropbox upload)
+
+Bonus points for:
+
+- Checking if the bug exists under latest experimental build
+- Checking if it is OS specific
+
+The OS and CDDA version are very important - with the pace of changes here it is possible the bug you have encountered has already been fixed. After that reproducibility is the key, so write your report with all the necessary details.
+
+### Enhancements and addition ideas
+
+We have hundreds of issues open - most of them are ideas and suggestions. Please search first if maybe something like what you have on mind has been already proposed. If so feel free to join the discussion! If you think your idea is related but sufficiently different - open a new issue and perhaps refer to the older discussion (use GitHub's `#issue_number` reference system).
+
+Remember to take part in the discussion of your suggestions.
+
```

Comment:
True but I'm all for centralizing things. Getting up to speed on this has been a nightmare because of how scattered all the information is - and most of it may only live in people's heads.